### ISSUE TITLE
Can't load `/smui.css` in app.html ?

### ISSUE THREAD

**Issue by kuchaguangjie** [hperrin/svelte-material-ui#489]
**Describe the bug**
After upgrade sveltekit, which use vite 3.0, when open page, get error:
```
Failed to load url /smui.css?direct (resolved id: /smui.css?direct).
This file is in /public and will be copied as-is during build without going through the plugin transforms,
and therefore should not be imported from source code. It can only be referenced via HTML tags.
```

Under `static/`, I can see `smui.css` and `smui-dark.css` generated, it used to work, but suddenly got above issue.

Link: [stackoverflow question](https://stackoverflow.com/questions/73077205)

**Comment by kuchaguangjie** [comment]
I've figured it out, need to use `/public` dir for asserts, instead of `/static` dir.

**Comment by wickning1** [comment]
I think this is a very recent bug in sveltekit:

https://github.com/sveltejs/kit/issues/5672

**Comment by hperrin** [comment]
@wickning1 It seems to be. I'm hitting the same issue in the demo site.

**Comment by kuchaguangjie** [comment]
bug fixed in [1.0.0-next.392](https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%401.0.0-next.392)
